src/main/java/org/prebid/server/bidder/tappx/TappxBidder.java

```diff
@@ -142,10 +137,8 @@ private static BidType getBidType(String impId, List<Imp> imps) {
         BidType bidType = BidType.banner;
```

Comment:
Since you've used a `return BidType.video;` in `if (imp.getVideo() != null)` block, there's no point in declaring `bidType` variable if you are not going to re-assign it and use it for return.

Please remove this variable declaration and instead `return BidType.banner;` in the end of the method.